package.json

```diff
@@ -33,9 +33,9 @@
   },
   "license": "ISC",
   "devDependencies": {
-    "cheerio": "^0.22.0",
     "grunt": "^1.0.1",
     "grunt-cli": "^1.2.0",
+    "material-colors-object": "^1.0.0",
```

Comment:
I think we should have a strictly specific version `1.0.0` here because the content of `material-colors` now depends on the exact version of `material-colors-object`. Otherwise, we cannot build the same content of `material-colors` when you publish a new version of `material-colors-object` like `1.1.0` or `1.0.1` with a slightly different content.

Reply:
Added 2d7266b to fix this.